### DISCUSSION TITLE
HomeSpan example using NVS

### DISCUSSION THREAD

**Discussion by prof-miles0** [HomeSpan/HomeSpan#72] [Ideas]
Hi!

It would be great to have an easy to understand example on how to use the NVS to store the current state of a switch, window cover... in case the board restarts. I tried to use your somfy code but it is quite difficult to understand.

Thanks

**Comment by HomeSpan** [comment]
That's a good idea.  The SOMFY example is actually quite complicated and the NVS logic is more complex than typically needed to store switch settings, etc.  Will create a cleaner example in an upcoming release.
